llvm/test/Transforms/LICM/variant-aainfo.ll

```diff
@@ -0,0 +1,66 @@
+; RUN: opt < %s -passes=aa-eval -evaluate-aa-metadata -aa-pipeline=basic-aa,tbaa -print-all-alias-modref-info -disable-output 2>&1 | FileCheck --check-prefix=CHECKAA %s
+; RUN: opt < %s -S -passes=licm | FileCheck --check-prefix=CHECKLICM %s
```

Comment:
Use update_test_checks.py.

Reply:
I will look into to it, but out of curiosity: why do you mandate generated assertions? From https://llvm.org/docs/TestingGuide.html#generating-assertions-in-regression-tests I understood generated assertions are optional, if too tedious to write manually. Is this to update the test in the future?

Reply:
See https://llvm.org/docs/TestingGuide.html#best-practices-for-regression-tests for basic testing guidelines.

Reply:
Fixed in commit 7fd5d3cd328d.